### DISCUSSION TITLE
Decimal / Hexdecimal Switch in the Bookmarker

### DISCUSSION THREAD

**Discussion by SporyTike** [RetroAchievements/RAIntegration#952] [Ideas]
When adding addresses to the bookmarker it always automatically sets the display format based on the setting set in the achievement editor. So when "Show Decimal" is enabled, new bookmarks always have the format "Dec" by default and when it's disabled they have the format "Hex".
When doing some RAM digging and having the wrong format set in the achievement editor you always need to open it just to change the default type of new bookmarks.

In my opinion the bookmarker and achievement editor should not share the same format preference since just because I enable Decimal in the editor to make my life easier creating stuff like score achievements or Measured bars doesn't necessarily mean I want to investigate all addresses in Decimal format.

And even in case they continue to use the same preference there should at least be an option in the bookmarker itself to allow developers to switch the preference without needing to do some extra steps by opening another toolkit window.
![image](https://user-images.githubusercontent.com/45672834/210160576-8154f1b3-35bf-4e6f-bbe3-1f5d4936627c.png)

**Comment by Jamiras** [comment]
The format (and size) can be changed after they've been added to the bookmark window by clicking on them.

**Reply by SporyTike** [reply]
I am aware of that, it's not a big problem. I just feel like this would be a little QoL update. In my opinion it's just a bit annoying that when my last achievement was coded with Decimal format and I then search for new addresses like pointers I either need to set the format to Hex manually everytime when having the address added in the bookmarker or needing to switch to the achievement editor to remove the checkmark. I would like to have an opinion in the bookmarker itself to change the default just to save some time. If you think it's not worth it, feel free to close this discussion.
